dev/io.openliberty.faces.4.0.thirdparty/bnd.bnd

```diff
@@ -29,8 +29,7 @@ Export-Package: \
   org.apache.myfaces.config;version=${Implementation-Version}, \
   org.apache.myfaces.renderkit;version=${Implementation-Version}, \
   org.apache.myfaces.renderkit.html.base;version=${Implementation-Version}, \
-  org.apache.myfaces.renderkit.html.util;version=${Implementation-Version}, \
-  org.apache.myfaces.core.api.shared;version=${Implementation-Version}
```

Comment:
`org.apache.myfaces.core.api.shared` should be discussed later on as I think it should be exposed by the API instead.

Reply:
@volosied  it should already be the case: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/io.openliberty.jakarta.faces.4.0/bnd.bnd#L24

Reply:
I added a comment for us to review this further in:  #22204